pegasus/sites.v3/code.org/public/css/page/video-page-styles.scss

```diff
@@ -0,0 +1,262 @@
+
+@import 'color', 'font', 'typography', 'breakpoints', 'barlow-semi-condensed-font';
+
+.hero-banner-basic {
```

Comment:
The design of the hero banner changed a bit — I left [comments in Figma](https://www.figma.com/file/p20ItesAK1Xs48indzeHlf/Video-Page?type=design&node-id=218%3A169&mode=design&t=M93I8ax7pZsrBVAB-1), but let me know if you have any questions.